home/forms.py

```diff
@@ -19,3 +20,11 @@ class SignUpForm(UserCreationForm):
     class Meta:
         model = User
         fields = ('username', 'first_name', 'last_name', 'email', 'gender', 'password1', 'password2', )
+
+
+class CommentForm(forms.ModelForm):
+    content = RichTextField(blank=True, null=True)
```

Comment:
Should this be the `RichTextBleackFeild` too? Also id expect to see `| safe` somewhere as a result of this change.

Reply:
I changed it, 
the important thing is that `RichTextBleackField ` will be on the model , because there I preform the validation. 
the  `RichTextBleackField ` is same as `RichTextField` but with the validations.
and there is `|safe` on the question_detail.html ( `<p>{{ answer.content | safe }}</p>` )